styles.css

```diff
@@ -0,0 +1,954 @@
+  font-weight: 900;
+  padding-top: 12px;
+  color: #f03824;
+}
+
+.fSpeakers {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 44px 0;
+  font-size: 18px;
+}
+
+.fSpeakers hr {
+  width: 10%;
+  border: 1px solid rgb(247, 65, 37);
+  margin-top: 10px;
+}
+
+.desktopSpeakers {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-items: center;
+  align-items: center;
+  gap: 5px;
+  padding: 0 24px 24px 24px;
+  position: relative;
+}
+
+.desktopSpeakers hr {
+  width: 10%;
+  margin: 5px 0;
+}
+
+.forex {
+  margin: 0 24px 24px 24px;
+}
+
+.forexClass2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 200px;
+  margin: 5px auto;
+  background-color: #ec5242;
+  opacity: 0.9;
+  position: relative;
+  color: white;
+  font-size: 24px;
+  font-weight: 700;
+}
+
+.forexClass1::before {
+  content: '';
+  position: absolute;
+  background: url(./images/learn-Forex1.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  height: 200px;
+  opacity: 0.3;
+}
+
+.forexClass1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 200px;
+  margin: 15px auto;
+  background-color: #ec5242;
+  opacity: 0.9;
+  position: relative;
+  color: white;
+  font-size: 24px;
+  font-weight: 700;
+}
+
+.forexClass2::before {
+  content: '';
+  position: absolute;
+  background: url(./images/learn-Forex1.jpeg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  height: 200px;
+  opacity: 0.3;
+}
+
+@media screen and (min-width: 768px) {
+  .fa-brands {
+    color: white;
+  }
+
+  .dskNav1 {
+    display: block;
+    position: relative;
+    font-weight: 700;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
+  .dskNavUl1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    background-color: #545454;
+    color: #d3d3d3;
+    padding: 20px 150px 20px 20px;
+    gap: 34px;
+  }
+
+  .dskNavUl1 a {
+    text-decoration: none;
+    color: white;
+  }
+
+  .dskNav2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    background-color: #fff;
+  }
+
+  .classes {
+    border: 4px solid rgb(246, 85, 85);
+    padding: 5px 15px;
+  }
+
+  .dskNavUl2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    color: #7b7b7b;
+    font-size: 18px;
+    padding-right: 150px;
+  }
+
+  .dskNavUl2 li {
+    margin-left: 22px;
+  }
+
+  .dskNavUl2 a {
+    text-decoration: none;
+    color: rgb(126, 126, 126);
+  }
+
+  .logoImg {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: none;
+    padding: 10px 20px 15px 60px;
+  }
+
+  .aboutLogo {
+    background-color: #fff;
+  }
+
+  .footerContainerLogo {
+    background-color: #fff;
+    margin: 0;
+    padding: 0;
+  }
+
+  .logoImg img {
+    width: 30%;
+  }
+
```

Comment:
- [ Optional ] Please consider using the default `width` of your `logo` on the menu, currently, the image is too big which makes the menu occupy a lot of space than it should. This is an optional suggestion but is highly recommended. Please see the screenshot below

## Behance design
![Screenshot from 2023-02-11 07-01-30](https://user-images.githubusercontent.com/16392046/218238169-a1a201a9-a9f7-426e-a797-575e7cd30426.png)

## Your design
![Screenshot from 2023-02-11 06-52-31](https://user-images.githubusercontent.com/16392046/218238047-0d91d2c4-1f93-4586-a761-42e7a3c62dfb.png)